### DISCUSSION TITLE
알림을 웹소켓으로 사용하셨는데 왜 그러셨나요?

### DISCUSSION THREAD

**Discussion by heejae101** [heejae101/heejae101.github.io#6] [Q&A] [answered]
설명해주세요.

**Comment by yunJeong3** [comment, marked as answer]
http는 server에 request 요청 시에만 서버에서 response를 반환하는데, 웹소켓을 사용하면 서버에 웹소켓이 허용되면 웹소켓을 종료할 때까지 계속 실시간으로 요청을 받기 때문에 사용했어요
